recipe/meta.yaml

```diff
@@ -1,54 +1,53 @@
-{% set version = "1.4.0" %}
+{% set version = "2.5.6" %}
 
 package:
   name: conda-lock
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/conda-lock/conda_lock-{{ version }}.tar.gz
-  sha256: 55c115a852e63395db8a6a90cf2892aab0543959f2d0bafb83158b6863831b8a
+  sha256: 323db8b6438b46648c4ba4dab81547851ee8266dbe913680f7837c94cc5f1023
 
 build:
   number: 0
-  skip: True  # [py<36]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - conda-lock = conda_lock:main
 
 requirements:
   host:
     - python
-    - hatchling
-    - hatch-requirements-txt
-    - hatch-vcs
     - pip
     - setuptools
     - wheel
+    - hatchling
+    - hatch-vcs
   run:
     - python
-    - cachecontrol-with-filecache >=0.12.9
-    - cachy >=0.3.0
     - click >=8.0
```

Comment:
Not a blocker, but maybe adding some of the same comments and/or listing poetry 1.1.15 explicitly would be nice. https://github.com/conda/conda-lock/blob/v2.5.6/pyproject.toml#L44